src/Controller/PathHandler.py

```diff
@@ -41,3 +74,13 @@ def data_path(relative_path):
         for name in files:
             if name == relative_path:
                 return os.path.join(root, name)
+
+def database_path() -> Path:
+    """
+    To get the path of the database.
+    This path was set up in
+    `Configuration.Configuration.__init__`
+
+    :return: Path
+    """
+    return Path(os.environ['USER_ONKODICOM_HIDDEN']).joinpath('OnkoDICOM.db')
```

Comment:
is it possible that the environment variable will not be set?  And if so, will this give the desired result?